docker-compose.yml

```diff
@@ -1,9 +1,26 @@
 version: "3"
 
 services:
+  base:
+    image: atb00ker/ready-to-run:openwisp-base
+    build:
+      context: build
+      dockerfile: openwisp_base/Dockerfile
+      args:
+        DASHBOARD_APP_PORT: $DASHBOARD_APP_PORT
+    deploy:
+      replicas: 1
+    environment:
+      - DB_ENGINE=$DB_GIS_ENGINE
+    env_file:
+      - .env
+    volumes:
+      - openwisp_static:/opt/openwisp/static
+      - openwisp_media:/opt/openwisp/media
+
```

Comment:
Are the build/args, deploy and volumes syntax needed for the base container?  It looks as though its intended to be a running container vs a base for building other containers.

Reply:
oh yeah, I also use `Makefile` a lot, it's a shame I did not start using this in OpenWISP.
If we can use a `Makefile` to initialize the build, let's do it, so we centralize the logic in one source of truth.

Reply:
> Are the build/args, deploy and volumes syntax needed for the base container?

Oops, no they are not required at all.

Reply:
Please review the Makefile. :-)